src/structs/pdb.rs

```diff
@@ -468,7 +465,7 @@ impl<'a> PDB {
     /// Find all hierarchies matching the given search. For more details see [Search].
     /// ```
     /// use pdbtbx::*;
-    /// let (pdb, errors) = open_pdb("example-pdbs/1ubq.pdb", StrictnessLevel::Loose).unwrap();
+    /// let (pdb, errors) = open_pdb_with_options("example-pdbs/1ubq.pdb", ReadOptions::new().set_level(StrictnessLevel::Loose)).unwrap();
```

Comment:
Change this from `open_pdb_with_options` to the `ReadOptions::read()`

Reply:
or just make it use `open` assuming that the file is ok with the default strictness